src/model_theory/fraisse.lean

```diff
@@ -253,9 +253,9 @@ variables {L} (K)
 
 /-- A structure `M` is a Fraïssé limit for a class `K` if it is countably generated,
 ultrahomogeneous, and has age `K`. -/
-structure is_fraisse_limit [countable_functions L] : Prop :=
+structure is_fraisse_limit [countable (Σl, L.functions l)] : Prop :=
```

Comment:
If you `@[protect_proj]` this structure, then you can use `_root_.countable` without `_root_.` prefix.